packages/react/src/components/DotcomShell/DotcomShell.js

```diff
@@ -7,10 +7,12 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import { settings } from 'carbon-components';
+import { settings } from '@carbon/ibmdotcom-utilities';
+import { settings as carbonSettings } from 'carbon-components';
 import { Masthead, Footer } from '@carbon/ibmdotcom-react';
 
 const { prefix } = settings;
+const cPrefix = carbonSettings.prefix;
```

Comment:
Again, my destructuring syntax knowledge is a little rusty, but could you do this?

```javascript
const { prefix: cPrefix} = carbonSettings;
```

Reply:
We can get rid of this actually, since there won't be a naming conflict now.